src/index.html

```diff
@@ -1,13 +1,435 @@
-    <title>Miami</title>
+    <title>THE MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/logo.svg"
+      type="image/x-icon"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+
+            <a href="/"class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.svg"
+                alt="MET"
+              >
+            </a>
+            <div class="header__wrapper">
+              <a href="tel:+1 212-535-7710"class="icon icon--phone header__wrapper__call">
+              </a>
+
+              <a href="#menu" class=" icon icon--menu header__wrapper__menu-opener"></a>
+            </div>
+          </div>
+
+          <h1 class="header__title">
+            Welcome<br>to The MET
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu" >
+      <div class="container">
+        <div class="menu__content">
+          <div class=" menu__top top-actions">
+            <a href="/"class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.svg"
+                alt="MET"
+              >
+            </a>
+            <div class="header__wrapper">
+              <a href="/" class=" icon icon--cross header__wrapper__menu-opener"></a>
+            </div>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#hours" class="menu__link">Museum hours</a>
+            </li>
+            <li class="menu__item">
+              <a href="#now-on-view" class="menu__link">Now on view</a>
+            </li>
+            <li class="menu__item">
+              <a href="#explore" class="menu__link">More to Explore Online</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contact us</a>
+            </li>
+          </ul>
+          <p class="menu__label">The Met Fifth Avenue</p>
+          <a href="tel:+1 212-535-7710" class="menu__call menu__call--phone">+1 212-535-7710</a>
+          <p class="menu__label">The Met Cloisters</p>
+          <a href="tel:+1 212-923-3700" class="menu__call menu__call--phone--two">+1 212-923-3700</a>
+          <a href="tel:+1 212-535-7710" class="menu__call menu__call--action">call to order</a>
```

Comment:
Add hover effect to these elements:
![image](https://user-images.githubusercontent.com/37161315/170512755-911e0dda-e9af-465a-ad16-92313e366104.png)